docs/index.md

```diff
@@ -334,6 +334,17 @@ describe('my suite', () => {
 
 _If you do not need to use_ Mocha's context, lambdas should work. Be aware that using lambdas will be more painful to refactor if the need eventually arises!
 
+Alternatively, you can override certain context variables, such as test timeouts, by chain-calling methods of the created tests and/or hooks:
+
+```js
+describe('my suite', () => {
+  beforeEach(() => {}).timeout(1000);
+  it('my test', () => {
+    assert.ok(true);
+  }).timeout(1000); // should set the timeout of this test to 1000 ms; instead will fail
```

Comment:
I don't see a need for the comment; I think it's been established by the added sentence.

```suggestion
  }).timeout(1000);
```